### DISCUSSION TITLE
Differences in responsive breakpoints in the documentation

### DISCUSSION THREAD

**Discussion by Ihsanasfari** [tailwindlabs/tailwindcss#14125] [Help] [answered]
I installed Tailwind CSS using npm with version "^3.4.3". When I wanted to use responsive features, it was different from the documentation. For example, the '2xl' breakpoint is supposed to have a minimum width of 1536px, but when I used it in my React project, '2xl' had a minimum width of 1320px. I am using the default Tailwind breakpoints without making any changes to the tailwind.config.js.

Is there an update to the Tailwind CSS documentation at [this link](https://tailwindcss.com/docs/responsive-design), or if I'm wrong, please let me know?

![Screenshot 2024-08-05 151644](https://github.com/user-attachments/assets/abbb8f7b-a81b-41ec-9708-b1aeea4f6e6e)
![image](https://github.com/user-attachments/assets/cbae3704-4709-4808-bc22-d10c01924e8b)

**Comment by RobinMalfait** [comment]
Hey!

The default value for `2xl` is still `1536px`. You can see the full default config here: https://github.com/tailwindlabs/tailwindcss/blob/1f23c2e8421cf8f674cf4dfe9940e51ff21f3d4a/stubs/config.full.js#L820

If you have the value `1320px`, then you are probably overwriting it in your Tailwind config. What does your Tailwind config look like?

**Reply by Ihsanasfari** [reply]
im not overwritting the '2xl'

this is my tailwind.config.js

![image](https://github.com/user-attachments/assets/d8ea078d-7cb2-4cdb-a10c-f1fbbd8f6132)

**Reply by adamwathan** [reply, marked as answer]
Hey! So you are overriding it, that’s part of what that `withMT` function is doing — it changes a bunch of defaults in the framework:

https://github.com/creativetimofficial/material-tailwind/blob/main/packages/material-tailwind-react/src/utils/withMT.js

Here are the custom breakpoints they’re using:

https://github.com/creativetimofficial/material-tailwind/blob/main/packages/material-tailwind-react/src/theme/base/breakpoints.js

**Reply by Ihsanasfari** [reply]
Ohhh, I see. Thank you very much for your help.
